modules/static/protocolbuffers/gofeatures/buf.yaml

```diff
@@ -0,0 +1,2 @@
+version: v1
+name: buf.build/protocolbuffers/gofeatures
```

Comment:
Do we want to name this `protobuf-go` instead of `gofeatures` in case they ever add more go-specific proto files in the future?

It could be confusing that wellknowntypes contains java/cpp features but the go ones are defined elsewhere.

Reply:
> Do we want to name this `protobuf-go` instead of `gofeatures`.

🤷. I don't anticipate anything else being added here. This is a core Protobuf runtime, so I wouldn't expect it to need any definitions other than custom options and features related to Go code gen, in which case "gofeatures" seems fine. The upside to this name is that it is more descriptive as to what it actually contains. The downside, as you mentioned, is if they eventually add other stuff to it, suddenly making `gofeatures` a bit of a misnomer.

> It could be confusing that wellknowntypes contains java/cpp features but the go ones are defined elsewhere.

Yep. But that's the state of the world with the authoritative sources, too: the core protoc distribution does _not_ include the `go_features.proto` file. The Protobuf team said it was intentional: since `protoc-gen-go` is a separate/independent plugin from the core languages implemented by `protoc`, they are using the approach that any other plugin author would need to take to provide feature definitions. This is much like any other plugin, where the sources for custom options, etc, must be distributed in parallel to the plugin itself.

Also, this file is revisioned with the protobuf-go repo, not with the main protoc releases, so trying to bundle it inside the wellknowntypes module would be problematic from that perspective (not to mention the fact that the current fetch mechanism is not flexible enough to accommodate a single module whose contents are pulled from multiple repos).

FWIW, the Protobuf team only intend to include features for languages implemented by `protoc` in the core release (and thus mirrored into our `protocolbuffers/wellknowntypes` module).

Reply:
I think on balance, I think we keep it as `gofeatures`. This gets back to us choosing to have conflicting namespaces between plugins and modules, I choice I regret. If we wanted to name this consistently, we'd actually just name it `protocolbuffers/go`, but that's not possible since that is taken by the plugin. What we should really do is come up with a naming scheme for repos that share (or could share) plugins and modules, which will be common in the case of options.

With all that typed out...would be open to such a scheme. For example, `buf.build/foo/bar` and `buf.build/foo/bar-api` is one option, but I don't like that `-api` is redundant.